src/HttpHeader.h

```diff
@@ -118,8 +118,8 @@ class HttpHeader
     bool hasNamed(const SBuf &s, String *value = 0) const;
     /// \deprecated use SBuf method instead.
     bool hasNamed(const char *name, unsigned int namelen, String *value = 0) const;
-    String getByNameListMember(const char *name, const char *member, const char separator) const;
-    String getListMember(Http::HdrType id, const char *member, const char separator) const;
+    SBuf getByNameListMember(const char *name, const char *member, const char separator) const;
+    SBuf getByIdListMember(Http::HdrType id, const char *member, const char separator) const;
```

Comment:
I do not think we should rename this method: "By ID" is the standard/fast way of addressing header fields so we do not add `ById` to method names (with delById() being an exception/mistake possibly caused by `delete` being a reserved keyword).

"By name", on the other hand, is slow and we should bring the reader's attention to that fact (using "Slowly" may have been better).

```suggestion
    SBuf getListMember(Http::HdrType id, const char *member, const char separator) const;
```